zookeeper-client/zookeeper-client-c/src/zookeeper.c

```diff
@@ -1457,7 +1457,7 @@ zhandle_t *zookeeper_init_ssl(const char *host, const char *cert, watcher_fn wat
     zcert.ca = strtok(strdup(cert), ",");
```

Comment:
This sounds like a memory leak. I find only `free` of field `certstr` but not `ca` in [`destroy`](https://github.com/apache/zookeeper/blob/90f813ea38a85ff2715662bad75f9bb6387fe4a6/zookeeper-client/zookeeper-client-c/src/zookeeper.c#L692). @symat @ztzg @anmolnar